x-pack/plugins/security_solution/public/onboarding/components/onboarding_body/cards/alerts/types.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { CardSelectorListItem } from '../common/card_selector_list';
+import type { CardAssetType } from '../types';
+
+export enum AlertsCardItemId {
+  list = 'list',
+  timeline = 'timeline',
+  analyzer = 'analyzer',
+  sessionView = 'session_view',
+}
+
+export type AlertsCardSelectorListItem = CardSelectorListItem & CardAssetType;
```

Comment:
We are duplicating the same type 3 times. Can we define it only once in the generic _types.ts_ file:
```
export type CardSelectorAssetListItem = CardSelectorListItem & CardAsset;
```

Reply:
a more elaborated explanation: https://github.com/elastic/kibana/pull/199311#discussion_r1856993075